src/config.rs

```diff
@@ -1,358 +1,1994 @@
 use webrtc_audio_processing_sys as ffi;
 
-pub use ffi::InitializationConfig;
-
 #[cfg(feature = "derive_serde")]
 use serde::{Deserialize, Serialize};
 
-/// A level of non-linear suppression during AEC (aka NLP).
+/// A configuration for initializing a Processor instance.
+#[derive(Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "derive_serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "derive_serde", serde(default))]
+pub struct InitializationConfig {
+    /// Number of the input and output channels for the capture frame.
```

Comment:
Wouldn't "capture" imply only "input"?

```suggestion
    /// Number of input channels for the capture frame.
```